.github/workflows/main.yml

```diff
@@ -0,0 +1,125 @@
+name: Kafka Demo Test 
```

Comment:
Since we will likely re-use these workflows for other demos, we should keeps the steps/names a generic as possible. So I would call this "test_kafka.yml" and then create a generic reusable workflow that this test calls (call it test_example.yml) similar  to the structure of the ACI examples repo where test_key_release https://github.com/microsoft/confidential-aci-examples/blob/main/.github/workflows/test_key_release.yml uses the generic test_example https://github.com/microsoft/confidential-aci-examples/blob/main/.github/workflows/test_example.yml

Reply:
ok